src/client/src/_store/ducks/count.ts

```diff
@@ -0,0 +1,32 @@
+import { AnyAction } from 'redux';
+
+// types & constants
+const INCREMENT: string = 'valo-lineups/counter/increment';
+const DECREMENT: string = 'valo-lineups/counter/decrement';
```

Comment:
rename to 'valo-lineups/count/increment' etc

Reply:
updated!